matcher/dashboard/views/imports.py

```diff
@@ -14,7 +15,7 @@
 
 from ..forms.imports import EditImport, UploadImport
 
-__all__ = ["ImportFileListView", "ShowImportFileView"]
+__all__ = ["ImportFileListView", "ShowImportFileView", "DownloadImportFileView" ]
```

Comment:
Kinda ironic that this extra space after `"DownloadImportFileView"` not only is not PEP-8 on its own, but makes you cross the 80 characters threshold which is also not PEP-8.

Reply:
Also, for consistency with `exports`, Download should come before Show, both in this list and for the function definitions.

Reply:
done

Reply:
Cheers.